pkg/client/portforward/portforward_test.go

```diff
@@ -253,6 +249,20 @@ func TestGetListener(t *testing.T) {
 		},
 	}
 
+	// On some linux systems, ::1 does not resolve to localhost but to localhost6 or
+	// ip6-localhost. To make the test case portable, we need to do a reverse lookup on ::1 and
+	// trying to bind a port with the name.
+	names, err := net.LookupAddr("::1")
+	if err == nil && len(names) > 0 {
```

Comment:
Hi Fabio,

That's not really a skip, nor a failure.

The test case wil try to bind an IPv6 by name. If no name is associated 
to ::1, the test does not run.

The cases of failure would be if in case of a failed lookup, and the 
test will


Le 22/04/15 22:40, Fabio Yeon a écrit :
>
> In pkg/client/portforward/portforward_test.go 
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7187#discussion_r28912507>:
>
> > @@ -253,6 +249,20 @@ func TestGetListener(t *testing.T) {
> >  		},
> >  	}
> >
> > +	// On some linux systems, ::1 does not resolve to localhost but to localhost6 or
> > +	// ip6-localhost. To make the test case portable, we need to do a reverse lookup on ::1 and
> > +	// trying to bind a port with the name.
> > +	names, err := net.LookupAddr("::1")
> > +	if err == nil && len(names) > 0 {
>
> Silent skip on a failure seems bad. Why not fail or at least log on a 
> failed lookup?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7187/files#r28912507>.
>